lbry/lbry/dht/protocol/routing_table.py

```diff
@@ -175,6 +175,17 @@ def __init__(self, loop: asyncio.AbstractEventLoop, peer_manager: 'PeerManager',
             )
         ]
 
+    def initialize_buckets_from_ranges(self, ranges: typing.List[typing.Tuple[int, int]]):
+        if len(self.buckets) > 1:
+            raise RuntimeError('Routing table already in use')
+        if len(ranges) == 0:
+            return
+        self.buckets: typing.List[KBucket] = [
+            KBucket(
+                self._peer_manager, range_min=ri[0], range_max=ri[1], node_id=self._parent_node_id
+            ) for ri in ranges
+        ]
+
```

Comment:
this can be in `__init__`, then the new function isn't necessary:


```
self._buckets = [
    KBucket(
        self._peer_manager, range_min, range_max, self._parent_node_id
    ) for range_min, range_max in bucket_ranges or [(0, 2**constants.hash_bits)]
]
```

Reply:
Yes, I initially put it here when I was doing validation of restored buckets (which I decided to move to db and then we abandonded persisting buckets) and didn't want to have logic / exceptions in the constructor, and forgot to remove it...